src/ui/server/helpers/has-policy-contact-changed/index.test.ts

```diff
@@ -45,7 +45,7 @@ describe('server/helpers/is-policy-contact-changed', () => {
     });
   });
 
-  describe(`when application ${IS_SAME_AS_OWNER} is true and body ${IS_SAME_AS_OWNER} is set to true`, () => {
+  describe(`when application ${IS_SAME_AS_OWNER} is false and body ${IS_SAME_AS_OWNER} is set to true`, () => {
```

Comment:
This mentions "application IS_SAME_AS_OWNER" but in the test it's called `mockContact`, should one of these change?

Reply:
should be `when policyContact's ${IS_SAME_AS_OWNER}`